src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php

```diff
@@ -209,4 +218,55 @@ private function getBundle($path)
 
         return $bundle;
     }
+
+    /**
+     * @dataProvider provideCompletionSuggestions
+     */
+    public function testComplete(array $input, array $expectedSuggestions)
+    {
+        $extractedMessagesWithDomains = [
+            'messages' => [
+                'foo' => 'foo',
+            ],
+            'validators' => [
+                'foo' => 'foo',
+            ],
+            'custom_domain' => [
+                'foo' => 'foo',
+            ],
+        ];
+        $extractor = $this->createMock(ExtractorInterface::class);
+        $extractor
+            ->expects($this->any())
+            ->method('extract')
+            ->willReturnCallback(
+                function ($path, $catalogue) use ($extractedMessagesWithDomains) {
+                    foreach ($extractedMessagesWithDomains as $domain => $message) {
+                        $catalogue->add($message, $domain);
+                    }
+                }
+            );
+
+        $tester = new CommandCompletionTester($this->createCommand([], [], null, [], [], $extractor, [new ExtensionWithoutConfigTestBundle()], ['fr', 'nl']));
+        $suggestions = $tester->complete($input);
+        $this->assertSame($expectedSuggestions, $suggestions);
+    }
+
+    public function provideCompletionSuggestions()
+    {
+        yield 'locale' => [
+            [''],
+            ['fr', 'nl'],
+        ];
+
+        yield 'bundle' => [
+            ['fr', '--domain', 'messages', ''],
+            ['ExtensionWithoutConfigTestBundle', 'extension_without_config_test'],
```

Comment:
I get an exception when I use the bundle_name in snake case. Does it work for you? (It's ok with BundleName)

```console
$ bin/console debug:translation en framework 

                                                                          
  "framework/translations" is neither an enabled bundle nor a directory.  
                                                                          
```

Reply:
Taking a look ASAP 👍